dlfcn_test.go

```diff
@@ -40,7 +27,7 @@ func getSystemLibrary() (string, error) {
 func TestRegisterFunc(t *testing.T) {
 	library, err := getSystemLibrary()
 	if err != nil {
-		t.Errorf("couldn't get system library: %s", err)
+		t.Skipf("couldn't get system library: %s", err)
```

Comment:
Now we have a go:build restriction, this should never come. So Errorf is fine (sorry for going back and forth)

Reply:
Confirmed this is now Errorf, thanks